libr/core/visual.c

```diff
@@ -254,7 +254,7 @@ static bool __core_visual_gogo (RCore *core, int ch) {
 				break;
 			}
 			(void)p->consbind.get_size (&scr_rows);
-			int scols = r_config_get_i (core->config, "hex.cols");
+			size_t scols = r_config_get_i (core->config, "hex.cols");
```

Comment:
`r_config_get_i` returns a `ut64`, not `size_t`

Reply:
size_t >= ut64 i believe modern C should use size_t when there is a choice.

Reply:
No size_t is 32-bit on i386. Try `gcc -m32 a.c` to print sizeof(size_t).

Reply:
what @eagleoflqj says is true, moreover even just for consistency if a function returns `ut64` you should use the same type if possible.

Reply:
Okay, it is fixed